packages/concerto-cto/lib/printer.js

```diff
@@ -386,9 +386,17 @@ function toCTO(metaModel) {
             case `${MetaModelNamespace}.ImportAllFrom`:
                 result += `\nimport ${imp.namespace}.*`;
                 break;
-            case `${MetaModelNamespace}.ImportTypes`:
-                result += `\nimport ${imp.namespace}.{${imp.types.join(',')}}`;
+            case `${MetaModelNamespace}.ImportTypes`:{
+                let aliasedTypes=new Map();
+                if(Object.prototype.hasOwnProperty.call(imp,'aliasedTypes')){
```

Comment:
Is there a good reason not to do the following here?

```suggestion
                if(imp.aliasedTypes){
```

Reply:
I took this suggestion from eslint documentation, and yes, we can achieve the same as you suggested. 

But the difference in both is as follows:

1. **Object.prototype.hasOwnProperty.call(imp, 'aliasedTypes')**: This checks if the object imp has the property aliasedTypes, regardless of whether it is undefined or null. 
2. **imp.aliasedTypes:** This checks if the property aliasedTypes on the **object imp is true** (i.e., not undefined, null, 0, false, NaN, or an empty string).

In our case, we pass an array of objects as value to the field `aliasedTypes` so both would work the same